index.js

```diff
@@ -9,6 +9,7 @@ var syncRequest = require('sync-request')
 var strformat = require('strformat')
 var through = require("through2")
 var observable = require("riot-observable")
+var slash = require("slash")
```

Comment:
Please also add the respective dependency in package.json

Reply:
Ups, yes, I committed the change.